### ISSUE TITLE
[suggestion] Please add compality with LethalLevelLoader

### ISSUE THREAD

**Issue by SpazzMaticus107** [ceelery/Kast#7]
Please move this moon from LE to LLL. Lethal Expansion is restrictive as hell, incompatible with many mods and you have to build your modpack around it. It sucks big time. In case you haven't heard yet. 
API in question: https://thunderstore.io/c/lethal-company/p/IAmBatby/LethalLevelLoader/

**Comment by ceelery** [comment]
Agreed, this is currently my main focus before adding more to the map itself. Making good progress, but it will take some time
